apps/chat/src/store/chat/chat.epics.ts

```diff
@@ -57,4 +71,43 @@ const setFormValueEpic: AppEpic = (action$, state$) =>
     }),
   );
 
-export const ChatEpics = combineEpics(setFormValueEpic);
+const getConfigurationSchemaEpic: AppEpic = (action$, state$) =>
+  action$.pipe(
+    filter(ChatActions.getConfigurationSchema.match),
+    switchMap(({ payload }) => {
+      const selectedConversations =
+        ConversationsSelectors.selectSelectedConversations(state$.value);
+      const savedConfigurationSchema =
+        selectedConversations[0]?.messages?.[0]?.custom_content
+          ?.configuration_schema;
+
+      if (savedConfigurationSchema) {
+        return of(
+          ChatActions.getConfigurationSchemaSuccess(savedConfigurationSchema),
+        );
+      }
+
+      return ApplicationService.getConfigurationSchema(payload.modelId).pipe(
+        switchMap((schema) => {
+          return of(ChatActions.getConfigurationSchemaSuccess(schema));
+        }),
+        catchError(() => {
+          return of(ChatActions.getConfigurationSchemaFailed());
+        }),
+      );
+    }),
+  );
+
+const getConfigurationSchemaFailedEpic: AppEpic = (action$) =>
+  action$.pipe(
+    filter(ChatActions.getConfigurationSchemaFailed.match),
+    map(() => {
+      return UIActions.showErrorToast('Failed to load chat starters');
```

Comment:
We should use 'translate' function for the Toast texts
```suggestion
      return UIActions.showErrorToast(translate('Failed to load chat starters'));
```